conftest.py

```diff
@@ -0,0 +1,24 @@
+# This code is part of OpenFE and is licensed under the MIT license.
```

Comment:
This is ugly but the only way we can appropriately call pytest at the root level directory and have additional options picked up (otherwise we'd have to call pytest openfe.tests and pyest openfecli.tests separately).

Interestingly, this is the recommended approach by ptyest: https://docs.pytest.org/en/latest/how-to/writing_plugins.html#plugin-discovery-order-at-tool-startup :/ ... doesn't make it any prettier though

Reply:
@mikemhenry does bring up the good point that this will probably not get packaged if it's in root, which would break the idea of doing this anyways. Shall we move it back to openfe.tests?

Reply:
I vote we move back to `openfe.tests` so they get packaged. Then we can tell users to run `pytest --pyargs openfecli.tests --pyargs openfe.tests -v -n auto` and that will test everything. Then if we add more burn in tests that we want people to run, they can add the `--runslow` arg

Reply:
And then on our main CI we just need to add `openfe/tests` to the pytest invocation (if we are not using the pyargs method)

Reply:
pytest --pyargs invocation won't work with the flag it seems (this is a known issue), my suggestion is that we either a) turn the flag into `--fast` so that by default we already run all the tests, or b) switch to using env variables (boring, but at least it'll likely work?)